packages/story-editor/src/components/publishModal/content/storyPreview.js

```diff
@@ -298,6 +311,7 @@ const StoryPreview = () => {
                 {(hasUploadMediaAction || enablePosterHotlinking) && (
                   <StyledMedia
                     onChange={handleChangePoster}
+                    onDelete={onDelete}
```

Comment:
Just curious, wasn't deleting purposefully not added to the publishing modal? Since it's required for publishing.

Reply:
https://github.com/GoogleForCreators/web-stories-wp/pull/11941#discussion_r925324309